modules/assim.batch/R/pda.load.data.R

```diff
@@ -38,13 +40,19 @@ load.pda.data <- function(settings, con) {
     
     vars.used.index <- which(format$vars$bety_name %in% c(inputs[[i]]$variable.name))
     
+    if(format$file_name == "AmeriFlux.level2.h.nc"){
+      time.row <- NULL
+    }else{
+      time.row <- which(format$vars$bety_name %in% c("year", "month", "week", "day", "hour", "minute", "second"))
+    }
```

Comment:
Oh, OK, I see. You're referring to these lines right?

```
    st <- format$vars$storage_type
    time.row <- which(nchar(st)>1 & substr(st, 1,1) == "%")
```

 Makes sense. It was in calc.benchmak, my workflow doesn't go through calc.benchmark but I'll add the lines in `pda.load.data`